site/src/_layouts/default.hbs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>d3fc {{#if title}}:{{/if}} {{title}}</title>
+    <meta name="description" content="A collection of components that make it easy to build interactive charts with D3.">
+
+    <script src="{{assets}}/scripts.js"></script>
+
+    <link rel="stylesheet" href="{{assets}}/styles.css" />
+  </head>
```

Comment:
Good point, although, I don't think SVG is mentioned on the homepage or the index page for the components - so it may not be immediately obvious to users who don't know much about D3/d3fc that the library uses SVG.